src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +173,16 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_ops(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
```

Comment:
Here please check if no arguments were passed using `ov::js::validate`

Reply:
hey @almilosz, 

Is it necessary to check using `ov::js::validate`?
I thing we can also check validation directly in function like this-
```suggestion
    Napi::Env env = info.Env();
    
    if (info.Length() != 0) {
        reportError(env, "Invalid argument. Expected no arguments.");
        return env.Undefined();
    }
```

Reply:
hey @almilosz, 

Is it necessary to check using `ov::js::validate`?
I thing we can also check validation directly in function like this-
```suggestion
    Napi::Env env = info.Env();
    
    if (info.Length() != 0) {
        reportError(env, "Invalid argument. Expected no arguments.");
        return env.Undefined();
    }
```

Reply:
It will work but one of the purpose of using `ov::js::validate` is to ensure that error messages are consistent across different locations